packages/google_atlas/lib/src/utils/lat_lng_bounds_utils.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:atlas/atlas.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart' as GoogleMaps;
+
+class LatLngBoundsUtils {
+  static GoogleMaps.LatLngBounds mapMarkersToLatLngBounds(Set<Marker> markers) {
+    double maxNorth = -90.0;
+    double maxEast = -180;
+    double minSouth = 90.0;
+    double minWest = 180.0;
+
+    if (markers == null || markers.isEmpty) {
+      return null;
+    }
+
+    markers.forEach((marker) {
+      if (marker.position.latitude > maxNorth) {
+        maxNorth = marker.position.latitude;
+      }
+      if (marker.position.latitude < minSouth) {
+        minSouth = marker.position.latitude;
+      }
+      if (marker.position.longitude > maxEast) {
+        maxEast = marker.position.longitude;
+      }
+      if (marker.position.longitude < minWest) {
+        minWest = marker.position.longitude;
+      }
+    });
+
+    // take the shorter half around the globe
+    if ((maxEast - minWest) > 180.0) {
+      final temp = maxEast;
+      maxEast = minWest;
+      minWest = temp;
+    }
+
+    return GoogleMaps.LatLngBounds(
+      northeast: GoogleMaps.LatLng(maxNorth, maxEast),
+      southwest: GoogleMaps.LatLng(minSouth, minWest),
+    );
+  }
+
+  static GoogleMaps.LatLngBounds newLatLngBoundsFromPosition(
+    GoogleMaps.LatLngBounds originalBounds,
+    LatLng position,
+  ) {
+    if (originalBounds == null) {
+      return null;
+    }
+    if (position == null) {
+      return originalBounds;
+    }
+    double maxNorth = originalBounds.northeast.latitude;
```

Comment:
It is not needed to use `double` here, since the compiler will infer the type from the assigned expression... but I think it would be useful to use `var` for these variables, since their values is mutable within this method